nf_core/pipeline-template/.github/workflows/cloud_tests_small.yml

```diff
@@ -0,0 +1,76 @@
+name: small-sized tests on cloud providers
+run-name: Submitting workflow to all cloud providers using small sized data
+on:
+  workflow_dispatch:
+    inputs:
+      platform:
+        description: "Platform to run test"
+        required: true
+        default: "all"
+        type: choice
+        options:
+          - all
+          - aws
+          - azure
+          - gcp
+jobs:
+  run-small-tests-on-aws:
+    if: ${{ github.event.inputs.platform == 'all' || github.event.inputs.platform == 'aws' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: seqeralabs/action-tower-launch@v2
+        with:
+          workspace_id: ${{ secrets.TOWER_WORKSPACE_ID }}
+          access_token: ${{ secrets.TOWER_ACCESS_TOKEN }}
+          compute_env: ${{ secrets.TOWER_CE_AWS_CPU }}
+          workdir: "${{ secrets.TOWER_BUCKET_AWS }}/work/{{ short_name }}/{% raw %}work-${{ github.sha }}"
```

Comment:
does this path have to start with `s3://`?

Reply:
The assumption is that the `TOWER_BUCKET_xyz` env vars at the nf-core organization level would be used to source the exact location.

This also gives use the freedom to test `/fusion` based paths on these different platforms.